pom.xml

```diff
@@ -5,7 +5,7 @@
     <groupId>org.nuxeo.ecm.distribution</groupId>
     <artifactId>nuxeo-distribution</artifactId>
     <!-- greater or equal to runtime HF required version -->
-    <version>10.10-HF39</version>
+    <version>10.10-HF44</version>
```

Comment:
Upgrade this version only if needed for compile or runtime: the version stated here is used as a minimal requirement for the Marketplace packages.